wrapper_test.go

```diff
@@ -253,13 +297,19 @@ func (w *wrapperTestSuite) TestLambdaHandlerE2ELocal() {
 			assert.Equal(w.T(), string(inputPayload), lumigoEnd.Event)
 			assert.Equal(w.T(), version, lumigoStart.SpanInfo.TracerVersion.Version)
 
+			if lumigoStart.LambdaType == "http" {
+				assert.Equal(w.T(), 200, lumigoStart.SpanInfo.HttpInfo.Response.StatusCode)
+				assert.Equal(w.T(), `{\"response\": \"test\"}`, lumigoStart.SpanInfo.HttpInfo.Response.Body)
+				assert.Equal(w.T(), `{\"Agent\": \"test\"}`, lumigoStart.SpanInfo.HttpInfo.Response.Headers)
+			}
```

Comment:
Why this block isn't with the other checks above?

Reply:
this is for http what do you mean?

Reply:
![image](https://user-images.githubusercontent.com/49228804/160277806-775b9726-79b1-41fe-b8b0-24b5aa95d11c.png)
I meant that we have the same `if` twice, shouldn't all the checks of the HTTP spans be in one place?

Reply:
aahh the one is for the start span right and the other one is for end span

Reply:
But it's not related to start\end spans (those are function-spans), it's about http-spans.

Reply:
nope there is also http span in it, you missed the new entry in the test cases